Code/Support/RestKit-Prefix.pch

```diff
@@ -5,6 +5,11 @@
 #ifdef __OBJC__
     #import <Foundation/Foundation.h>
 
+// Fix for compilation on iOS 6 where this is not defined.
+#if !defined NS_DESIGNATED_INITIALIZER
```

Comment:
http://stackoverflow.com/questions/8617712/difference-between-ifndef-and-if-defined-in-c <http://stackoverflow.com/questions/8617712/difference-between-ifndef-and-if-defined-in-c>

> On Jun 17, 2015, at 3:20 PM, Samuel E. Giddins <notifications@github.com> wrote:
> 
> In Code/Support/RestKit-Prefix.pch <https://github.com/RestKit/RestKit/pull/2238#discussion_r32681558>:
> 
> > @@ -5,6 +5,11 @@
> >  #ifdef __OBJC__
> >      #import <Foundation/Foundation.h>
> >  
> > +// Fix for compilation on iOS 6 where this is not defined.
> > +#if !defined NS_DESIGNATED_INITIALIZER
> #ifndef ?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/RestKit/RestKit/pull/2238/files#r32681558>.
>